examples/00-EDB/03_5G_antenna_example_parametrics.py

```diff
@@ -102,29 +93,24 @@ def points(self):
 edb.stackup.add_layer("TOP", "Substrat")
 
 ###############################################################################
-# Create linear array
-# ~~~~~~~~~~~~~~~~~~~
-# Create the first patch of the linear array.
-
-
+# Create the the first patch and feed line using the ``Patch``, ``Line``classes defined above.
```

Comment:
```suggestion
# Create the first patch and feed line using the ``Patch`` and ``Line`` classes defined earlier.
```
What is "feed line" here--is it being using as a verb? Or, is it noun, in which case it should be "feedline" instead.